shipyard_blueprints/shipyard-bp-utils/shipyard_bp_utils/files.py

```diff
@@ -260,8 +262,8 @@ def find_all_file_matches(file_names: list, file_name_re: str) -> list:
     """
     matching_file_names = [file for file in file_names if re.search(file_name_re, file)]
 
-    logger.info(f"Found {len(matching_file_names)} file matches.")
```

Comment:
I think this line that prints out the file matches should be left in the `info` level. That is useful information to the users that should be available outside of `debug` mode